app/models/BannerTests.scala

```diff
@@ -44,9 +44,9 @@ case class BannerTest(
     deviceType: Option[DeviceType] = None,
     campaignName: Option[String] = Some("NOT_IN_CAMPAIGN"),
     signedInStatus: Option[SignedInStatus] = Some(SignedInStatus.All),
-    isBanditTest: Option[Boolean] = None,
     consentStatus: Option[ConsentStatus] = Some(ConsentStatus.All),
     deploySchedule: Option[BannerTestDeploySchedule] = None,
+    methodologies: List[Methodology] = Nil
```

Comment:
although the `methodologies` is mandatory, it's defaulted to `Nil` (an empty list), meaning validation does not fail when fetching existing data from dynamodb

Reply:
In general I try to avoid `Option[List[T]]`, because you have the following possible states:
1. `None`
2. empty list
3. non-empty list
And 1 and 2 sort of mean the same thing!
If we make the field optional then the client has to handle that possibility as well.

Reply:
ahhh good point!